EarlyBird/CoursesTableViewController.swift

```diff
@@ -24,4 +28,57 @@ class CoursesTableViewController: UITableViewController {
         cell!.textLabel!.text = courseNames[indexPath.row]
         return cell!
     }
+    
+    func makeGetRequest() {
+        // Set up the URL request for one specific course to before tring all courses
+        let courseEndpoint: String = "https://early-bird-courses.herokuapp.com/api/v1/courses/1"
```

Comment:
Thanks @ryanflach Yeah, I have been declaring the variable / constant type in most places to start and then I try removing it to see if I get a compiling error.